File: java/carolruo/FootballPoints.java
public class FootballPoints {

    public static void main(String[] args) {
        System.out.println("wins: 1, draws: 2, losses: 3. Total points: " + footballPoints(1, 2, 3));
        System.out.println("wins: 5, draws: 5, losses: 5. Total points: " + footballPoints(5, 5, 5));
        System.out.println("wins: 1, draws: 0, losses: 0. Total points: " + footballPoints(1, 0, 0));
    }

    public static int footballPoints(int wins, int draws, int losses) {
        return (wins * 3 + draws);
    }
}

File: javascript/guilhermescr/DesafioDois/footballPoints.js
function footballPoints(wins, draws, losses) {
  let points = wins * 3 + draws * 1 + losses * 0;
  return console.log(points);
}

footballPoints(3, 4, 2);
footballPoints(5, 0, 2);
footballPoints(0, 0, 1);
